src/main/java/seedu/address/logic/parser/UpdateEventCommandParser.java

```diff
@@ -0,0 +1,59 @@
+package seedu.address.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DESCRIPTION;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_REPEATABLE_DATE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_REPEATABLE_INTERVAL;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_UPDATE_INDEX;
+
+import java.time.LocalDate;
+import java.util.stream.Stream;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.UpdateEventCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.task.Interval;
+import seedu.address.model.task.repeatable.Event;
+
+/**
+ * Parses input arguments and creates a new UpdateEventCommand object
```

Comment:
End sentence with period, class names should be in `{@code}` to standardise with other comments as much as possible.